packages/sdk/src/types.ts

```diff
@@ -1037,6 +1069,21 @@ interface AccountInfoCommon {
     accountEncryptedAmount: AccountEncryptedAmount;
     accountReleaseSchedule: AccountReleaseSchedule;
     accountCredentials: Record<number, AccountCredential>;
+    /**
+     * The stake on the account that is in cooldown.
+     * There can be multiple amounts in cooldown that expire at different times.
+     * This was introduced in protocol version 7, and so is not present in
+     * earlier protocol versions.
```

Comment:
**Suggestion (non-blocking):** The wording makes it sound like it will be undefined prior to P7, but I think a more precise wording would be that: it will always be the empty array for any block validated prior to P7.

Reply:
I copied it from the docs in the GRPC proto types, but you're right. Good catch 😄